### PR TITLE
Problem: Binaries with latest patches of cosmos-sdk and ibc-go are not released

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Upgraded `cosmos-sdk` to `v0.44.6` and `ibc-go` to `v2.0.3`

### CODE REVIEW

**Review by tomtau — CHANGES_REQUESTED** [review]
besides a version bump, a few pending items for the binary release:
 
- [x] swagger security upgrade
- [ ] changelog
- [ ] gomod2nix

File: go.sum [hunk 1]
```diff
@@ -194,15 +195,16 @@ github.com/coreos/pkg v0.0.0-20160727233714-3ac0863d7acf/go.mod h1:E3G3o1h8I7cfc
 github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f/go.mod h1:E3G3o1h8I7cfcXa63jLwjI0eiQQMgzzUDFVpN/nH/eA=
 github.com/cosmos/btcutil v1.0.4 h1:n7C2ngKXo7UC9gNyMNLbzqz7Asuf+7Qv4gnX/rOdQ44=
 github.com/cosmos/btcutil v1.0.4/go.mod h1:Ffqc8Hn6TJUdDgHBwIZLtrLQC1KdJ9jGJl/TvgUaxbU=
-github.com/cosmos/cosmos-sdk v0.44.5 h1:t5h+KPzZb0Zsag1RP1DCMQlyJyIQqJcqSPJrbUCDGHY=
 github.com/cosmos/cosmos-sdk v0.44.5/go.mod h1:maUA6m2TBxOJZkbwl0eRtEBgTX37kcaiOWU5t1HEGaY=
```
Comment (tomtau) [inline go.sum:198]: I guess 0.44.5 is brought in by ibc-go -- maybe adding `replace` to force depending on a single version?

**Comment by devashishdxt** [comment]
> * swagger security upgrade

This is done in https://github.com/crypto-org-chain/chain-main/pull/704

**Comment by tomtau** [comment]
maybe also the new version of Tendermint: https://github.com/tendermint/tendermint/releases/tag/v0.34.16 ?

**Comment by tomtau** [comment]
> > * swagger security upgrade
> 
> This is done in #704

ok, good to link that PR in a changelog

**Review by tomtau — APPROVED** [review]

**Pull request merged**